### PR TITLE
Upgrade OpenAI Dependency (to fix "Quota Reached" error) & some lint fixes

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: .github/workflows/prodution-deploy.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - openai-version-bump
```
Comment (cmyui) [inline .github/workflows/prodution-deploy.yml]:
```suggestion
```

File: requirements.txt [hunk 2]
```diff
@@ -1,7 +1,6 @@
 asyncpg
-backoff
```
Comment (cmyui) [inline requirements.txt:2]: not bothering to maintain

Comment (cmyui) [reply requirements.txt:2]: in a follow-up pr i think i will try/catch the whole process & return an err msg to the discord if anything goes wrong

File: app/main.py [hunk 3]
```diff
@@ -212,22 +207,32 @@ async def on_message(message: discord.Message):
         )
         if not gpt_response:
             await message.channel.send(
-                f"Request failed after multiple retries.\n"
-                f"Please try again after some time.\n"
-                f"If this issue persists, please contact cmyui#0425 on discord."
+                "Request failed after multiple retries.\n"
+                "Please try again after some time.\n"
+                "If this issue persists, please contact cmyui#0425 on discord."
```
Comment (cmyui) [inline app/main.py]:
```suggestion
                'If this issue persists, please contact "cmyui" on discord.'
```

**Review by tsunyoku — DISMISSED** [review]

**Review by infernalfire72 — DISMISSED** [review]

**Pull request merged**